### ISSUE TITLE
Missing GL_ constants

### ISSUE THREAD

**Issue by dkorpel** [BindBC/bindbc-opengl#16]
I've stumbled across some GL_ constants nowhere to be found in bindbc-opengl:
```D
// added in 1.5
enum GL_SRC1_ALPHA = 0x8589;

// added in 3.0
enum GL_VERTEX_ARRAY_BINDING = 0x85B5;

// added in 3.3
enum GL_RGB10_A2UI = 0x906F;

// added in 4.1
enum GL_RGB565 = 0x8D62;
enum GL_SHADER_BINARY_FORMATS = 0x8DF8;
enum GL_NUM_SHADER_BINARY_FORMATS = 0x8DF9;
```
http://www.opengl.org/registry/api/GL/glcorearb.h

I can simply make a PR adding these, but there might be more, and I wonder if there is an underlying reason. Were they simply overlooked? Was a different reference used for bindbc?

**Comment by mdparker** [comment]
I have no idea why they're missing. Possibly overlooked. I'll add them in.

**Comment by mdparker** [comment]
Fixed in 0.11.0. Looks like I simply overlooked them. `GL_NUM_SHADER_BINARY_FORMATS` was actually already there. Still seeing an error?

**Comment by dkorpel** [comment]
Thank you, this is solved now.
